tests/World/test_constants.py

```diff
@@ -1,5 +1,14 @@
 from adr.World import constants
+import numpy.testing as npt
 
 
 def test_air_molar_mass():
-    pass
+    assert constants.air_molar_mass == 0.02896
+
+
+def test_gravitacional_acceleration():
+    assert constants.gravitational_acceleration == 9.80665
+
+
+def test_air_gas_constant():
+    npt.assert_almost_equal(constants.air_gas_constant, 287.101, decimal=3)
```

Comment:
Maybe for easing on dev time, we can have the tests of all fixed constants on one function, and the tests for the calculated ones on another. No need for a separate function on each constant.